Apps/WebClient/src/ClientApp/src/views/registration.vue

```diff
@@ -393,7 +391,7 @@ export default class RegistrationView extends Vue {
     private smsNumber: string = "";
 
     private isEmailChecked: boolean = true;
-    private isSMSNumberChecked: boolean = false;
+    private isSMSNumberChecked: boolean = true;
```

Comment:
just to be sure; should it check if its not filled?

Reply:
as designed